pkg/generators/builder/builder.go

```diff
@@ -100,27 +102,75 @@ func (b *BuilderPatternGenerator) Init(c *generator.Context, w io.Writer) error
 }
 
 func (b *BuilderPatternGenerator) GenerateType(c *generator.Context, t *types.Type, w io.Writer) error {
-	log.Debugf("Checking type: %s", t.Name.Name)
-	if !b.needsGeneration(t) {
-		return nil
-	}
```

Comment:
This is now handled in the `Filter` function. GenerateType will only be called for types enabled after Filter.